dc-cudami-server/dc-cudami-server-webapp/src/main/java/de/digitalcollections/cudami/server/controller/identifiable/entity/geo/location/GeoLocationController.java

```diff
@@ -107,12 +123,15 @@ public long count() {
         "/latest/geolocations/identifier"
       },
       produces = MediaType.APPLICATION_JSON_VALUE)
-  public ResponseEntity<GeoLocation> getByIdentifier(
+  public void getByIdentifier(
       @RequestParam(name = "namespace", required = true) String namespace,
-      @RequestParam(name = "id", required = true) String id)
+      @RequestParam(name = "id", required = true) String id,
+      HttpServletRequest request,
+      HttpServletResponse response)
       throws IdentifiableServiceException {
-    GeoLocation result = geoLocationService.getByIdentifier(namespace, id);
-    return new ResponseEntity<>(result, HttpStatus.OK);
+    response.setStatus(HttpStatus.MOVED_PERMANENTLY.value());
```

Comment:
```suggestion
    response.setStatus(HttpServletResponse.SC_MOVED_PERMANENTLY);
```
Hier und an anderen Stellen vllt. besser die ints aus `HttpServletResponse` nutzen (`response` selbst ist ja auch ein `HttpServletResponse`, das mit Spring selbst nichts zu tun hat)